MultiPage-Clone/css/main.css

```diff
@@ -1,3 +1,528 @@
+  text-align: center;
+}
+
+.blue-line {
+  background-color: var(--Royal-blue);
+  width: 20%;
+  height: 3px;
+  border-radius: 9999px;
+}
+.button {
+  display: flex;
+  flex-direction: row;
+  gap: 20px;
+  font-size: 28px;
+}
+
+.btn-card {
+  width: 150px;
+  height: 45px;
+  border-radius: 9999px;
+  border: none;
+  background-color: var(--Royal-blue);
+  color: var(--White);
+  font-weight: 600;
+}
+.empty {
+  border: 1.5px solid var(--Black);
+  color: var(--Black);
+  background-color: var(--Royal-blue), 0;
+}
+
+.empty:hover {
+  background-color: var(--Royal-blue);
+}
+.media-information {
+  height: 50px;
+  background-color: var(--White);
+  margin: 0px 0px 20px;
+  display: flex;
+  flex-direction: row;
+  gap: 20px;
+  align-items: center;
+  justify-content: center;
+}
+
+.media-icon {
+  width: 25px;
+}
+
+.section-p {
+  width: 80vw;
+  margin: 45px;
+}
+.main-header {
+  margin: 45px 0px 15px;
+  font: var(--font-header1);
+  font-size: 55px;
+}
+.second-hdr {
+  font: var(--font-header1);
+  font-size: 22px;
+  font-weight: 600;
+}
+.sec-information {
+  font: var(--font-header1);
+  font-weight: 300;
+  font-size: 18px;
+  margin: 20px 0px;
+}
+
+.section-cont {
+  padding-top: 9em;
+}
+.left-side-btn {
+  display: none;
+}
+
+.ftr-media-icon {
+  width: 15px;
+  margin-bottom: 20px;
+}
+
+@media screen and (min-width: 540px) {
+  main {
+    margin-top: 120px;
+    flex-direction: row;
+    justify-content: center;
+  }
+  .background-index {
+    background: linear-gradient(
+      90deg,
+      var(--Dark-beige) 40vw,
+      var(--White) 40vw
+    );
+    min-height: calc(100vh - 240px);
+  }
+  .background-dark {
+    min-height: calc(100vh - 240px);
+  }
+
+  .card-section {
+    width: 375px;
+  }
+
+  .section-p {
+    max-width: 375px;
+  }
+
+  .left-side-btn {
+    display: inline;
+  }
+  .side-btn {
+    display: none;
+  }
+
+  .second-hdr {
+    margin: 30px 0px;
+  }
+
+  .sec-information {
+    margin: 30px 0px;
+  }
+
+  .main-header {
+    font-size: 100px;
+  }
+
+  .in-card::before {
+    display: none;
+  }
+
+  .media-information {
+    margin-left: 40px;
+  }
+
+  .nav-links {
+    font-size: 1.5em;
+  }
+
+  .nav-links1 {
+    color: var(--Royal-blue);
+    font-size: 1.5em;
+    font-weight: 300;
+  }
+}
+
+@media screen and (min-width: 540px) {
+  .button {
+    position: absolute;
+    display: flex;
+    flex-direction: row;
+    gap: 20px;
+    margin-left: 28em;
+  }
+}
+
+/* footer */
+
+.footer-cont {
+  background-color: var(--White);
+  display: flex;
+  flex-direction: column;
+  padding: 30px;
+  margin-right: 70px;
+}
+.information-footer {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  flex-wrap: wrap;
+  gap: 30px;
+  justify-content: space-between;
+}
+
+.footer-hdr {
+  font: var(--font-header1);
+  font-size: 15px;
+  font-weight: 500;
+}
+.footer-p {
+  font: var(--font-header1);
+  font-size: 16px;
+  font-weight: 300;
+}
+
+@media screen and (min-width: 540px) {
```

Comment:
![image](https://github.com/CodeYourFuture/Module-HTML-CSS/assets/118762745/7773a71c-bf47-4cab-8314-fa087dd40eac)
 there is too much empty space here: look at the demo vc yours 
 
![image](https://github.com/CodeYourFuture/Module-HTML-CSS/assets/118762745/84eaf0fd-c284-4a3b-ba55-0810d8e12551)